### DISCUSSION TITLE
dependency problems and my fix

### DISCUSSION THREAD

**Discussion by G1enB1and** [neonbjb/tortoise-tts#394] [Ideas]
I spent 4 hours trying to get this to run on windows with a new anaconda and found finally got it to work with the following additions/substitutions.
python 3.8 (3.9 gave errors).

conda install -c cctbx202208 numpy
conda install -c main llvm
pip install pydantic==1.9.1

**Comment by pawprint** [comment]
THANK YOU!

Also need to update the requirements for 


tokenizers==0.11.1

**Comment by Lunar13737** [comment]
THANK YOU!!!

Works perfectly!
